imagedephi/models/rules_models.py

```diff
@@ -0,0 +1,41 @@
+import importlib.resources
+from typing import Literal
+
+from pydantic import BaseModel
+import yaml
+
+
+class Replace(BaseModel):
+    method: Literal["replace"]
+    new_value: str
+
+
+class Keep(BaseModel):
+    method: Literal["keep"]
+
+
+class Delete(BaseModel):
+    method: Literal["delete"]
```

Comment:
@brianhelba these would likely be removed. They were part of the attempt to use the discriminator, but unless we intend to make method a nested dict I don't think this structure will work for us.